react-native-libraries.json

```diff
@@ -10274,5 +10274,12 @@
     "ios": true,
     "android": true,
     "web": true
+  },
+  {
+    "githubUrl": "https://github.com/howljs/react-native-auto-route",
+    "npmPkg": "react-native-auto-route",
```

Comment:
Hey @howljs, thank you for adding new library to the directory! 👍 

```suggestion
```

Small nit, we can skip npm package name, since the name matches repository.

Reply:
@Simek I removed `npmPkg`